packages/quantic/cypress/integration/summary/summary-selectors.ts

```diff
@@ -0,0 +1,20 @@
+import {ComponentSelector, CypressSelector} from '../common-selectors';
+
+export const summaryComponent = 'c-quantic-summary';
+
+export interface SummarytSelector extends ComponentSelector {
+  text: () => CypressSelector;
+  range: () => CypressSelector;
+  total: () => CypressSelector;
+  query: () => CypressSelector;
+}
+
+export const SummarySelectors: SummarytSelector = {
+  get: () => cy.get(summaryComponent),
+
+  text: () =>
+    SummarySelectors.get().find('.slds-rich-text-editor__output span'),
+  range: () => SummarySelectors.text().find('b').first(),
```

Comment:
What would you think of adding some marker classes to make it easier to retrieve each element?

`.find('b').first()` --> `.find('summary__range')`
`.find('b').eq(1)` --> `.find('summary__total')`
`.find('b').eq(2)` --> `.find('summary__query')`